### PR TITLE
feat: add query for last version released

### PR DESCRIPTION
**Pull request by linuxbandit**

### CODE REVIEW

File: scripts/myaegeestatus.js [hunk 1]
```diff
@@ -77,4 +85,30 @@ module.exports = (robot) => {
...
+
+      // err & response status checking code here
+      if (err != null || response.headers['content-type'].indexOf('application/json') === -1) {
+        robot.logger.error(err)
+        return res.send("An error occurred, or the answer was not JSON :(")
+      }
+
+      try {
+        const data = JSON.parse(body)
+        let message = `Last released version of ${service} is ${data[data.length-1].ref.replace('refs/tags/', '')}`
```
Comment (serge1peshcoff) [inline scripts/myaegeestatus.js:105]: I also suggest you to use backticks for fancy formatting the same way I did for deploy, for `service` and `version`

Comment (linuxbandit) [reply scripts/myaegeestatus.js:105]: isn't it what I did here?

Comment (linuxbandit) [reply scripts/myaegeestatus.js:105]: ah i got it now

Comment (serge1peshcoff) [reply scripts/myaegeestatus.js:105]: also you don't override this message later, so you can use const (and it's better to)

**Review by serge1peshcoff — APPROVED** [review]

**Pull request merged**